### ISSUE TITLE
Warmup Schedueler is Not Linear warmup, Increase in Log curve, inscrease Too Fast for big model

### ISSUE THREAD

**Issue by BitVoyage** [microsoft/DeepSpeed#1298]
![image](https://user-images.githubusercontent.com/10386094/129052778-948a5ea5-c084-4259-9a70-37a8e9cadc63.png)

**Comment by BitVoyage** [comment]
Would DeepSpeed Support Linear Warmup ? Thank You

**Comment by tjruwase** [comment]
@BitVoyage, thanks for your question. Can you please give more details on what you need? We are currently finishing some changes that will make it easy to integrate existing torch learning rate schedulers into DeepSpeed training.

**Comment by skpig** [comment]
@tjruwase I have encountered the same issue as well. I hope the learning rate can increase linearly in Warmup Stage.

**Comment by skpig** [comment]
@tjruwase @stas00  Is the feature being implemented? If not, I'm willing to give it a try.

**Comment by tjruwase** [comment]
@skpig, please go ahead with your plans. This will be greatly appreciated.
